tutorials/best_practices/scenes_versus_scripts.rst

```diff
@@ -207,12 +207,29 @@ In the end, the best approach is to consider the following:
     .. code-tab:: gdscript GDScript
 
       # game.gd
-      class_name Game # extends Reference, so it won't show up in the node creation dialog
-      extends Reference
+      class_name Game # extends RefCounted, so it won't show up in the node creation dialog
+      extends RefCounted
 
       const MyScene = preload("my_scene.tscn")
 
       # main.gd
       extends Node
       func _ready():
           add_child(Game.MyScene.instantiate())
+
+    .. code-tab:: csharp
+
+      // Game.cs
+      public partial class Game : RefCounted
+      {
+          public static readonly PackedScene MyScene = (PackedScene)ResourceLoader.Load("MyScene.tscn");
```

Comment:
I think according to the guidelines this should be a property.

Also, we likely prefer `GD.Load` but I'm not so sure about this. I think we use both `ResourceLoader.Load` and `GD.Load` in the documentation, we should probably pick one and be consistent.

```suggestion
          public static PackedScene MyScene { get; } = GD.Load<PackedScene>("MyScene.tscn");
```

Reply:
I used the example above as template, should I change it there too?

Reply:
Yes, if you could :pray: